### DISCUSSION TITLE
询问关于WiFi功率与国家码的问题

### DISCUSSION THREAD

**Discussion by CCMeta** [immortalwrt/immortalwrt#1612] [Q&A]
请问现在immortal使用的驱动驱动中
是否有一套完整的MTK定义的符合大多数国家法规的功率国家码信道对照表

还是说删除了所有其他国家 只存在一套默认的功率

**Comment by science2468** [comment]
[这是链接](https://git.kernel.org/pub/scm/linux/kernel/git/wens/wireless-regdb.git/tree/db.txt)， [这是openwrt里关于WiFi的文档](https://openwrt.org/docs/guide-quick-start/basic_wifi)
![{70011E2F-A0CE-4396-A0D7-15DA5C87574A}](https://github.com/user-attachments/assets/cf7703b9-5ba0-4270-bbc4-171652524f93)

**Reply by CCMeta** [reply]
感谢回复!
我想确认就是默认源码中这套逻辑是已经实现了的 
还是说要依据 您这边发出来的 db.txt 依据源码开发策略.

**Reply by science2468** [reply]
你要是用WiFi，在WiFi界面的国家选择中国，信道可以在界面选择

**Reply by CCMeta** [reply]
是的 我知道
我想确认一下我选了JP 是不是功率就会符合日本的法规
我测试了看起来是不行的  我再看看了 谢谢
